### PR TITLE
refactor: Update to use W3C Issuer API

### LINKED ISSUE #203: Use new W3C Issuer API
Update the edge-sandbox issuer to use new W3C Issuer APIs

### PR DESCRIPTION
**Pull request by rolsonquadras**
closes #203 

Signed-off-by: Rolson Quadras <rolson.quadras@securekey.com>

### CODE REVIEW

File: pkg/restapi/issuer/operation/operations.go [hunk 1]
```diff
@@ -390,28 +396,69 @@ func (c *Operation) prepareCreateCredentialRequest(subject map[string]interface{
 	delete(subject, "updated_at")
 	delete(subject, "userid")
 
-	req := &createCredential{
-		Context: []string{credentialContext},
-		Subject: subject,
-		Type:    []string{"VerifiableCredential", info.Scope},
-		Profile: vcsProfile,
+	profileResponse, err := c.retrieveProfile(vcsProfile)
```
Comment (sandrask) [inline pkg/restapi/issuer/operation/operations.go:399]: Does edge-sandbox still create profiles as part of set-up? Is so it is weird to retrieve profile every single time we want to create a credential. We should cache profiles.

Comment (rolsonquadras) [reply pkg/restapi/issuer/operation/operations.go:399]: Yes, no changes there. And agree with the caching part in non-sandbox environments.

Comment (sandrask) [reply pkg/restapi/issuer/operation/operations.go:399]: We should do it in sandbox too, profile is not something that changes. Sandbox issuer owns profile changes anyway. You added retrieving profile in this PR, this was not there before.

Comment (sandrask) [reply pkg/restapi/issuer/operation/operations.go:399]: You can add it as separate task and someone can pick it up (if we get there). We should not be calling profile there.

**Pull request merged**